src/schema/artist/current.js

```diff
@@ -1,15 +1,21 @@
-import { GraphQLNonNull, GraphQLObjectType, GraphQLString, GraphQLUnionType } from "graphql"
+import {
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
 import Image from "schema/image"
 import { error } from "lib/loggers"
 import moment from "moment"
 import { exhibitionPeriod } from "lib/date"
-import { ShowType } from "../show";
-import { SaleType } from "../sale";
+import { ShowType } from "../show"
+import { SaleType } from "../sale"
+import { date as DateFormat } from "schema/fields/date"
 
 const UnderlyingCurrentEventType = new GraphQLUnionType({
   name: "UnderlyingCurrentEvent",
   types: [ShowType, SaleType],
-  resolveType: ({ __type }) => __type
```

Comment:
someones prettier isnt running!

Reply:
Could definitely be mine! 🙈 I’ve been experiencing some weirdness with it specifically in MP.